### PR TITLE
EDU-771 Avoid gaps in flex layout because of older Safaris

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-771

### CODE REVIEW

File: src/components/collapsible.less [hunk 1]
```diff
@@ -13,12 +13,10 @@
 }
 
 .Collapsible-header {
```
Comment (adrianaluchian) [inline src/components/collapsible.less:15]: The collapsible arrow is no longer aligned to the right, which I find looks not so good (especially when the title is omitted):

Screen Shot 2022-08-31 at 17 21 49

File: src/components/pages/room.less [hunk 2]
```diff
@@ -5,25 +5,22 @@
...
+  padding: 0 @edu-content-padding;
 }
 
-.RoomPage-subtitleGroup {
-  display: flex;
-  gap: 7px;
-  align-items: center;
+.RoomPage-leaveRoomLink {
+  .m-control-group;
+  padding-left: @edu-control-spacing-wide;
```
Comment (adrianaluchian) [inline src/components/pages/room.less:15]: The "leave room" link is no longer visible 🤔 

How it looks on Int:
Screen Shot 2022-08-31 at 17 29 09

How it looks locally:
Screen Shot 2022-08-31 at 17 30 02

Comment (ahelmberger) [reply src/components/pages/room.less:15]: Ha! I reverted the condition in code in order to see the link while developing and forgot to set it back. Thanks for spotting it, it's fixed now.

File: src/components/collapsible.less (hunk 1 above)
Comment (ahelmberger) [reply src/components/collapsible.less:15]: Nice catch! I haven't realized. It's fixed now.

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**